examples/tests/sdk_tests/util/env.test.w

```diff
@@ -28,4 +28,15 @@ test "use util from inflight" {
   // --inflight tryEnv--
   let noValue = util.Util.tryEnv(NOT_ACTUAL_ENV);
   assert(noValue == nil);
-}
\ No newline at end of file
+}
+
+/**
+ * setEnv()
+ */
+util.setEnv("FOO", "bar");
+assert(util.env("FOO") == "bar");
+ 
+test "set env from inflight" {
+  util.setEnv("FOO", "baz");
+  assert(util.env("FOO") == "baz");
+}
```

Comment:
It might be worth documenting this only applies to the current phase (e.g. setting an environment variable in preflight doesn't cause it to automatically pollute all cloud.Functions of the app)